postgres.go

```diff
@@ -11,12 +12,12 @@ import (
 )
 
 const postgresSchema = `
-CREATE TABLE IF NOT EXISTS prefixes (
+ALTER TABLE IF EXISTS prefixes RENAME TO prefixes_root;
```

Comment:
Is it possible to keep the tablename for the default namespace at it was ?

Reply:
Yes that should be easy enough to change

Reply:
Perfect

Reply:
Done